vignettes/epidatr.Rmd

```diff
@@ -128,24 +130,8 @@ pub_covidcast(
   geo_type = "state",
   time_type = "day",
   geo_values = c("pa", "ca", "fl"),
-  time_values = epirange(20210105, 20210410)
-)
-```
-
-We can also request data for a single location at a time, via the `geo_values` argument.
```

Comment:
I guess having this separate is a bit silly if you actually get that `c("pa")=="pa"`

Reply:
Exactly